### DISCUSSION TITLE
Question about the pagination props in the uselist hooks.

### DISCUSSION THREAD

**Discussion by self-transition** [refinedev/refine#5443] [Q&A] [answered]
### Is your feature request related to a problem? Please describe.

I have a question about the pagination props in the uselist hooks.  I don't pass in the pagination props, but [here](https://github.com/refinedev/refine/blob/67a98aeed8695fd59b3e01d17ec1954ba8f804c3/packages/core/src/definitions/helpers/handlePaginationParams/index.ts#L15) it's  fallback to server mode, limit 10, which I'm not sure if that's what's expected or not, but it seems to me that it should be set to off mode by default, not server mode.

### Describe the thing to improve

Changing the pagination mode to off by default, but this will probably be a breaking change for people who were using it.

**Comment by BatuhanW** [comment, marked as answer]
As you noticed, default pagination mode is server, you can set it to off.
